iot-e2e-tests/edge-e2e/src/main/java/glue/ModuleGlue.java

```diff
@@ -23,6 +23,7 @@
 import java.util.*;
 
 
+@SuppressWarnings("ALL")
```

Comment:
Suppressing all E2E warnings. Please let me know if there is a better way to suppress a whole project instead of suppressing files individually.

Reply:
Is it because there are many of these and don't want to fix as of now?

Reply:
Didn't know about "ALL" :) 👍

Reply:
Last week we decided that we're not going to address any test warnings for now as they are lower priority.